src/utils/extractOutputs.ts

```diff
@@ -1,5 +1,24 @@
 export type OutputMapper = Record<string, string>;
 
+export namespace OUTPUT_MAPPERS {
+    export enum REVIEW {
+      summary = 'SUMMARY',
+      findings = 'FINDINGS',
+      score = 'SCORE',
+    }
+  
+    export enum STEP {
+      thoughts = 'THOUGHTS',
+      action = 'ACTION',
+      ux = 'UX',
+      a11y = 'ACCESSIBILITY',
+    }
+  
+    export enum SUMMARY {
+      summary = 'SUMMARY',
+    }
+}
+
```

Comment:
This a general util not necessarily will be used only for Pilot so let's add more information to the Enum names:
- `REVIEW` → `PILOT_REVIEW_SECTION`
- `STEP` → `PILOT_STEP`
- `SUMMARY` → `PILOT_SUMMARY`